influxql/functions.go

```diff
@@ -211,29 +211,31 @@ func MapMean(itr Iterator) interface{} {
 
 	for _, k, v := itr.Next(); k != 0; _, k, v = itr.Next() {
 		out.Count++
-		out.Sum += v.(float64)
+		out.Mean += (v.(float64) - out.Mean) / float64(out.Count)
 	}
 	return out
 }
 
 type meanMapOutput struct {
 	Count int
```

Comment:
i don't have a strong feeling - only uncertainty - so i'll leave it as is :smiley: